crengine/src/lvstsheet.cpp

```diff
@@ -4435,7 +4435,7 @@ bool LVStyleSheet::parse( const char * str, bool higher_importance, lString16 co
             }
             // parse declaration
             LVCssDeclRef decl( new LVCssDeclaration );
-            if ( !decl->parse( str, higher_importance, _doc, codeBase ) )
+            if ( !decl->parse( str, (_doc != NULL) ? _doc->getDOMVersionRequested() : 0, higher_importance, _doc, codeBase ) )
```

Comment:
May be set `lUInt32 domVersionRequested = _doc != NULL) ? _doc->getDOMVersionRequested() : 0;` at top of the function to not have to re-compute it it as each new declaration.

Reply:
Ok.